### PR TITLE
Upgrade babel and webpack

### PR DESCRIPTION
**Pull request by AWare**
This updates babel and webpack because they're both quite old and this unblocks moving the frontend architecture forwards easier.

### CODE REVIEW

File: conf/webpack.conf.js [hunk 1]
```diff
@@ -14,36 +18,39 @@ module.exports = {
...
               exclude: '/app/',
               loader: 'html-loader'
             },
-            {
-              test: /\.json$/,
-              exclude: /node_modules/,
-              use: 'json-loader'
-            },
             {
                 test: /\.scss$/,
```
Comment (jonathonherbert) [inline conf/webpack.conf.js:35]: By changing this test to /\.s?css$/, can we avoid two CSS rules?

File: app/views/main.scala.html [hunk 2]
```diff
@@ -15,7 +15,7 @@
             padding-top: 72px;
         }
     </style>
-    <link rel="stylesheet" type="text/css" href="@routes.Assets.versioned("build/main.css")"/>
+    <link rel="stylesheet" type="text/css" href="@routes.Assets.versioned("build/app.css")"/>
```
Comment (jonathonherbert) [inline app/views/main.scala.html:18]: Why the change?

**Comment by jonathonherbert** [comment]
Works fine locally 🎉 , but the production configuration breaks JS with an 'unknown provider' error that looks like a DI minification problem.

File: conf/webpack.conf.js (hunk 1 above)
Comment (AWare) [reply conf/webpack.conf.js:35]: Yeah, let's scrap that extra ruke.

File: app/views/main.scala.html (hunk 2 above)
Comment (AWare) [reply app/views/main.scala.html:18]: This splits out the main and admin css into different files, whereas previously it did not.
